marlowe-cli/app/Main.hs

```diff
@@ -44,4 +44,3 @@ fromGit = Nothing
 #else
 fromGit = Just ((\v -> if null v then mempty else " @ " <> v) . unpack . strip $ pack $(gitRevFromGit))
 #endif
-
```

Comment:
FYI: the `gitRevFromGit` function is pretty much useless and we probably should remove it now or soon.